pkg/shell/credentials.jsx

```diff
@@ -316,14 +316,12 @@ const UnlockKey = ({ keyName, load, onClose }) => {
                        <Button variant='link' onClick={onClose}>{_("Cancel")}</Button>
                    </>
                }>
-            <>
+            <Form onSubmit={e => { e.preventDefault(); return false }} isHorizontal>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17221-20221130-173646-50f39aba-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)